dnscrypt-proxy/serversInfo.go

```diff
@@ -227,13 +227,26 @@ func (serversInfo *ServersInfo) refresh(proxy *Proxy) (int, error) {
 	registeredServers := make([]RegisteredServer, len(serversInfo.registeredServers))
 	copy(registeredServers, serversInfo.registeredServers)
 	serversInfo.RUnlock()
-	liveServers := 0
+
+	// Always have proxy.certRefreshConcurrency servers being refreshed in parallel
```

Comment:
Setting to 1 equals to 1 by 1.

Reply:
Do you mean that setting to 1 is serial? How do I change the comment?

Reply:
Yeah, one at a time. What about no comments?

Reply:
OK, that's fine, I will remove them